seeds/perl_seed.pl

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use Getopt::Long;
+use Pod::Usage;
+
+# Perl script seed.
+# Use this to get started quickly, and customize it to your needs.
+
+my $type = 'flower';
+my $man = 0;
+my $help = 0;
+my $water = 0;
+GetOptions( 'help|h' => \$help, 
+            'man' => \$man,
+            'water|w' => \$water,
+          );
+pod2usage(1) if $help;
+pod2usage(-exitstatus => 0, -verbose => 2) if $man;
+
+# Write your code below
+if ($water){
+  water_the_plant(${type}) ;
+}
+else {
+  plant_a_seed(${type});
+}
+
+sub plant_a_seed {
+  my $seed = shift;
```

Comment:
Can you use a named argument rather than `shift`? You defined `$type` up on line 10 but never used it.

Reply:
In fact "shift" get the first argument pass to function call.
At line 21 there is main program with function call and use of $type.

# Write your code below
if ($water){
  water_the_plant(${type}) ;
}
else {
  plant_a_seed(${type});
}

But i understand why you don't find $type. I wrote it ${type}. I change that in same time as i add usage in help message.